appveyor.yml

```diff
@@ -32,7 +32,7 @@ before_test:
   - ctest -V
 
 artifacts:
-  - path: OpenMEEG-*-x64--static.exe
+  - path: c:\projects\openmeeg\build\OpenMEEG-*-x64--static.exe
```

Comment:
First problem : appveyor artifacts doesn't support the complete path. So you have to change it as it was before